htdocs/compta/paiement.php

```diff
@@ -528,7 +528,7 @@ function callForResult(imgId)
         $sql = 'SELECT f.rowid as facid, f.facnumber, f.total_ttc, f.multicurrency_code, f.multicurrency_total_ttc, f.type,';
         $sql.= ' f.datef as df, f.fk_soc as socid, f.date_lim_reglement as dlr';
         $sql.= ' FROM '.MAIN_DB_PREFIX.'facture as f';
-		$sql.= ' WHERE f.entity IN ('.getEntity('facture', $conf->entity).')';
+		$sql.= ' WHERE f.entity IN ('.getEntity('invoice', $conf->entity).')';
```

Comment:
@atm-quentin 
can you replace getEntity('invoice', $conf->entity) with getEntity('invoice')

Reply:
sure